api/src/router/routes/github-webhook.js

```diff
@@ -101,6 +101,7 @@ router.post(
           break;
         case PR_ACTIONS.LABELED:
           // Obtain the labels assigned to the PR
+          // eslint-disable-next-line no-case-declarations
```

Comment:
why is this needed?

Reply:
the nested switch is quite large, for cleanliness it could be broken up into its own function (just a thought !)